src/applications/disability-benefits/all-claims/pages/prisonerOfWar.js

```diff
@@ -1,27 +1,39 @@
 import fullSchema from '../config/schema';
-
 import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
-
 import PeriodOfConfinement from '../components/PeriodOfConfinement';
+import { addCheckboxPerNewDisability } from '../utils';
 
 export const uiSchema = {
   'ui:title': 'Prisoner of War (POW)',
```

Comment:
Interesting. I always thought it was `PoW` with a lowercase `o`. :man_shrugging:

Reply:
I thought so as well, but the internet seems to disagree.